clients/wallet/src/horizon.rs

```diff
@@ -1,9 +1,14 @@
 use std::{str::FromStr, sync::Arc, time::Duration};
 
 use async_trait::async_trait;
-use futures::future;
+use futures::{
+	future::{self, ready},
+	TryFutureExt,
+};
+
 use parity_scale_codec::{Decode, Encode};
-use primitives::TextMemo;
+use primitives::{TextMemo, TransactionEnvelopeExt};
+use rand::seq::SliceRandom;
```

Comment:
It seems like this is not used, so we can remove this and the crate from Cargo.toml again I suppose

Reply:
the rand? i used it to determine the horizon url fallback

Reply:
Ah oh, I thought I checked the code and didn't see any references. But maybe I just missed the function that used that haha